ecosystem/sep-0006.md

```diff
@@ -61,6 +61,12 @@ Authorization: Bearer <JWT>
 
 It is highly recommended to require authentication for all endpoints other than `/info` (and optionally `/fee`).
 
+### Source and Destination Accounts
+
+Note that both the source account of a withdrawal payment and the destination account of a deposit can be different than the account authenticated via SEP-10. 
+
+This is particularly useful for clients who use a third-party custody provider for their users' funds held on Stellar. Using this flexibility, a client can use a Stellar public and private key managed using their own infrastructure to authenticate with anchor services while using the Stellar account managed by a custodian as the source and destination of users' transactions.
```

Comment:
I think we should drop this paragraph as it is a little too prescriptive and makes it sound like this is the only use case. I can think of a variety of cases where someone might have different authenticating and source/destination accounts. For example, an account holder might have multiple Stellar accounts and want to deposit and withdraw from any of them, but they authenticate with one account because they only need to have one account with the anchor.

Reply:
Thats a good use case, I hadn't thought of that. But I agree it is very prescriptive so I'll remove it.